pandas/tests/series/test_analytics.py

```diff
@@ -1211,10 +1211,15 @@ def test_idxmin(self):
         assert result == 1
 
     def test_numpy_argmin(self):
-        # argmin is aliased to idxmin
         data = np.random.randint(0, 11, size=10)
-        result = np.argmin(Series(data))
-        assert result == np.argmin(data)
+
+        with pytest.warns(FutureWarning):
```

Comment:
Use `tm.assert_produces_warning(FutureWarning)`.  We don't use `pytest` warning context manager.